server/config/feature_flag_configs/local.json

```diff
@@ -1,25 +1,33 @@
-[{
+[
+  {
     "name": "autocomplete",
     "enabled": true,
     "owner": "gmechali",
     "description": "Feature flag to enable place autocomplete."
-},
-{
+  },
+  {
     "name": "dev_place_experiment",
     "enabled": true,
     "owner": "gmechali",
     "description": "Feature flag to enable the V2 place page only for the experiment places."
-},
-{
+  },
+  {
     "name": "dev_place_ga",
     "enabled": false,
     "owner": "dwnoble",
     "description": "Feature flag to enable the V2 Place page to GA."
-},
-{
+  },
+  {
     "name": "scroll_to_top_button",
     "deprecated": true,
     "enabled": true,
     "owner": "gmechali",
     "description": "Feature flag to enable the scroll to top button on the explore and place pages."
-}]
\ No newline at end of file
+  },
+  {
+    "name": "biomed_nl_routes",
+    "enabled": false,
```

Comment:
I think we want this at least enabled for local?

Reply:
enabled for local and autopush